tests/mod.rs

```diff
@@ -33,6 +33,12 @@ struct Struct<A, B, C> {
 	pub c: C,
 }
 
+#[derive(Debug, PartialEq, Encode, Decode)]
+struct A {
+	#[codec(encoded_as = "parity_codec::U8")]
```

Comment:
I think it should be possible to leave away the `"` to make it:
`#[codec(encoded_as = parity_codec::U8)]`
But still it has some problems due to macros not being hygienic as always.